<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QDialog" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TesTrack</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>831</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 175, 110, 255));
}
#frame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 217, 0, 255), stop:1 rgba(255, 165, 92, 255));
border-radius: 20px;
}
QLineEdit {
    font: 8pt MS Shell Dlg 2;
    background-color: rgb(255, 255, 255);
    border-radius: 18px;
    padding-left: 10px;
    padding-right: 10px;
}
QPushButton {
	background-color: rgb(255, 226, 0);
    
	color: rgb(0, 0, 0);
    padding: 6px;
    margin-top: 10px;
    border-radius: 15px;
}
QLineEdit:focus{
    background-color: rgb(221, 221, 221);
}
QPushButton:hover,
QPushButton:clicked {
    background-color: rgb(221, 221, 221);
}</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>771</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_Button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>770</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить ответ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otvet_input">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>780</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="marker_Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>770</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Маркер</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../src/marker.png</normaloff>../src/marker.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
